clouddriver-kubernetes/src/main/groovy/com/netflix/spinnaker/clouddriver/kubernetes/v2/caching/view/provider/KubernetesV2ManifestProvider.java

```diff
@@ -127,19 +147,24 @@ public KubernetesV2Manifest getManifest(String account, String location, String
         .orElse(new ArrayList<>());
   }
 
-  private KubernetesV2Manifest fromCacheData(CacheData data, String account) {
+  private KubernetesV2Manifest fromCacheData(
+      CacheData data, String account, boolean includeEvents) {
     KubernetesManifest manifest = KubernetesCacheDataConverter.getManifest(data);
     String namespace = manifest.getNamespace();
     KubernetesKind kind = manifest.getKind();
     String key = data.getId();
 
     List<KubernetesManifest> events =
-        cacheUtils
-            .getTransitiveRelationship(
-                kind.toString(), Collections.singletonList(key), KubernetesKind.EVENT.toString())
-            .stream()
-            .map(KubernetesCacheDataConverter::getManifest)
-            .collect(Collectors.toList());
+        !includeEvents
```

Comment:
Same comment as above about checking the positive condition `includeEvents` rather than `!includeEvents`